src/platform/silabs/efr32/BUILD.gn

```diff
@@ -164,10 +164,12 @@ static_library("efr32") {
       "wifi/wifi_config.h",
     ]
 
+    if (chip_enable_wifi_ipv4) {
```

Comment:
this can be wrapped inside the `if (use_wf200 || use_rs9116 || use_SiWx917)` logic

Reply:
I have added in the same logic